designer/server/src/main/scala/pl/touk/nussknacker/ui/api/NodesApiEndpoints.scala

```diff
@@ -497,35 +742,112 @@ object NodesApiEndpoints {
 
   }
 
+  private val noScenarioExample: EndpointOutput.OneOfVariant[NoScenario] =
+    oneOfVariantFromMatchType(
+      NotFound,
+      plainBody[NoScenario]
+        .example(
+          Example.of(
+            summary = Some("No scenario {scenarioName} found"),
+            value = NoScenario(ProcessName("'example scenario'"))
+          )
+        )
+    )
+
+  private val malformedTypingResultExample: EndpointOutput.OneOfVariant[MalformedTypingResult] =
+    oneOfVariantFromMatchType(
+      BadRequest,
+      plainBody[MalformedTypingResult]
+        .example(
+          Example.of(
+            summary = Some("Malformed TypingResult sent in request"),
+            value = MalformedTypingResult(
+              "Couldn't decode value 'WrongType'. Allowed values: 'TypedUnion,TypedDict,TypedObjectTypingResult,TypedTaggedValue,TypedClass,TypedObjectWithValue,TypedNull,Unknown"
+            )
+          )
+        )
+    )
+
+  private val noProcessingTypeExample: EndpointOutput.OneOfVariant[NoProcessingType] =
+    oneOfVariantFromMatchType(
+      NotFound,
+      plainBody[NoProcessingType]
+        .example(
+          Example.of(
+            summary = Some("ProcessingType type: {processingType} not found"),
+            value = NoProcessingType("'processingType'")
+          )
+        )
+    )
+
 }
 
 object TypingDtoSchemas {
 
+  import pl.touk.nussknacker.engine.api.typed.TypingType
   import pl.touk.nussknacker.engine.api.typed.TypingType.TypingType
   import pl.touk.nussknacker.engine.api.typed.typing._
-  import pl.touk.nussknacker.engine.api.typed.TypingType
   import sttp.tapir.Schema.SName
   import sttp.tapir.SchemaType.SProductField
   import sttp.tapir.{FieldName, Schema, SchemaType}
 
-  implicit lazy val typingResult: Schema[TypingResult] = Schema.derived
-
-  implicit lazy val singleTypingResultSchema: Schema[SingleTypingResult] =
-    Schema.derived.hidden(true)
+  implicit lazy val typingResult: Schema[TypingResult] = {
+    Schema(
+      SchemaType.SCoproduct(
+        List(
+          unknownSchema.hidden(true),
```

Comment:
let's add a comment with an explanation of why we have to do it like this. Let's always document hacks